connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -174,6 +175,14 @@ public ConnectorStateInfo connectorStatus(String connectorName) {
         String url = endpointForResource(String.format("connectors/%s/status", connectorName));
         try {
             return mapper.readerFor(ConnectorStateInfo.class).readValue(executeGet(url));
+        } catch (ConnectRestException e) {
+            //  the connector doesn't exist in the cluster yet
+            if (e.statusCode() == Response.Status.NOT_FOUND.getStatusCode()) {
+                log.warn("Could not find connector '{}' in cluster.", connectorName);
+                return null;
+            } else {
+                throw e;
+            }
```

Comment:
Nit: why use an `else` block here? The `if` block always returns, so really no `else` is needed, right?